### PR TITLE
TC 06.2.1_02 Select the men's bottom section

### PR DESCRIPTION
**Pull request by LenaDik**
https://trello.com/c/ukgQQb1g/458-us-0622-men-bottom

### CODE REVIEW

File: tests/menBottoms.spec.js [hunk 1]
```diff
@@ -37,4 +37,13 @@ test.describe('menBottoms', () => {
     await expect (page.getByRole('heading', { name: 'Bottoms' })).toBeVisible();
     await expect(page).toHaveURL('https://magento.softwaretestingboard.com/men/bottoms-men.html');
   })
+
+  test("Verify the men's bottom section", async ({ page }) => {
+    await page.locator('a[href*="/men.html"]').hover();
+    await page.locator("a[href*='/bottoms-men.html']").click();
+
+    await expect(page.getByRole('menuitem', { name: ' Men' })).toHaveText('Men');
+    await expect(page).toHaveURL(BASE_URL + '/men/bottoms-men.html');
```
Comment (alenavolakhava) [inline tests/menBottoms.spec.js:46]: Playwright error: `ReferenceError: BASE_URL is not defined` and it fails. you need to define the variable before using it. something like this: `const BASE_URL = "https://example.com"`

**Review by alenavolakhava — CHANGES_REQUESTED** [review]
test fails due to an undeclared BaseURL variable, please get it fixed and i will take a look again

**Review by alfiyaGumar — APPROVED** [review]

File: tests/menBottoms.spec.js (hunk 1 above)
Comment (verafes) [reply tests/menBottoms.spec.js]: it has beforeEach with directions to the base URL: await page.goto('/'); So we do not need to put navigation in each test.

Comment (LenaDik) [reply tests/menBottoms.spec.js:48]: but in other test cases we use BASE_URL, maybe everything should be in the same style and I will use BASE_URL in my TC?

**Review by alfiyaGumar — APPROVED** [review]

**Review by alenavolakhava — APPROVED** [review]
the test passed
Screenshot 2024-04-22 at 7 32 57 PM

**Review by skatoy — COMMENTED** [review]
+

**Pull request merged**